Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -0,0 +1,378 @@
+//
+//  DetailngCodebaseViewController.swift
+//  Manito
+//
+//  Created by creohwan on 2022/11/03.
+//
+
+import UIKit
+import SnapKit
+
+class DetailingCodebaseViewController: BaseViewController {
+
+    // MARK: - property
+    
+    private lazy var titleLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 34)
+        return label
+    }()
+    private lazy var periodLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .grey002
+        label.font = .font(.regular, ofSize: 16)
+        return label
+    }()
+    private lazy var statusLabel: UILabel = {
+        var label = UILabel()
+        label.backgroundColor = .mainRed
+        label.layer.masksToBounds = true
+        label.layer.cornerRadius = 11
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 13)
+        label.textAlignment = .center
+        return label
+    }()
+    private var missionBackgroundView: UIView = {
+        var view = UIView()
+        view.backgroundColor = .darkGrey004
+        view.layer.borderWidth = 1
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    private let missionTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "오늘의 개별 미션"
+        label.textColor = .darkGrey002
```

Comment:
p1;

```suggestion
        label.textColor = .grey002
```

저희가 사용하는 grey 색상이 daryGrey 와 grey가 있어요 !
여기서 사용되는 색상은 다크 그레이가 아닌 그냥 그레이 입니다 !

Reply:
> p1;
> 
> 저희가 사용하는 grey 색상이 daryGrey 와 grey가 있어요 ! 여기서 사용되는 색상은 다크 그레이가 아닌 그냥 그레이 입니다 !

aed57d9 으악! 이런 실수를 하다니~! 죄송합니다~

Reply:
괜찮습니다💛